packages/e2e-tests/specs/editor/various/autosave.test.js

```diff
@@ -338,6 +302,42 @@ describe( 'autosave', () => {
 		expect( notice ).toContain( AUTOSAVE_NOTICE_REMOTE );
 	} );
 
+	it( 'should clear sessionStorage upon user logout', async () => {
```

Comment:
I guess this one shouldn't be part of this PR.

Reply:
Yes, I just pushed this to check CI on GH. I'll remove it as it's merged here: https://github.com/WordPress/gutenberg/pull/26416